unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/GltfContainer/GltfContainerCollidersHandler.cs

```diff
@@ -114,8 +114,17 @@ private void SetVisibleMeshesColliders(IReadOnlyCollection<Renderer> renderers)
 
             foreach (Renderer renderer in renderers)
             {
-                MeshFilter meshFilter = renderer.GetComponent<MeshFilter>();
-                Mesh colliderMesh = meshFilter ? meshFilter.sharedMesh : null;
+                Mesh colliderMesh = null;
+
+                if (renderer is SkinnedMeshRenderer skinnedMeshRenderer)
+                {
+                    colliderMesh = skinnedMeshRenderer.sharedMesh;
+                }
+                else
+                {
+                    MeshFilter meshFilter = renderer.GetComponent<MeshFilter>();
+                    colliderMesh = meshFilter ? meshFilter.sharedMesh : null;
+                }
```

Comment:
not supporting `SkinnedMeshRenderer` for colliders was intentional 
https://github.com/decentraland/adr/blob/main/content/ADR-215-gltfcomponent-sdk-component.md?plain=1#L139

Reply:
Then the UpdateCollidersOnGltfChanged test should be changed since the sharknado model is 100% skinned mesh renderers

Reply:
thks ❤️